_analyzers/token-filters/keyword-repeat.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: default
+title: Keyword repeat
+parent: Token filters
+nav_order: 210
+---
+
+# Keyword repeat token filter
+
+The `keyword_repeat` token filter outputs the keyword version of a token into the token stream. This filter is typically used when you want to retain both the original token and its modified version after further token transformations, such as stemming or synonym expansion. The duplicated tokens allow the original, unchanged version of the token to remain in the final analysis alongside the modified versions.
+
```

Comment:
Line 10: Should "outputs" be "emits" (as in index.md)?

Reply:
Either way is fine; I'll change for consistency.